closure/goog/color/alpha.js

```diff
@@ -69,16 +69,18 @@ goog.color.alpha.hexToRgbaStyle = function(hexColor) {
 
 
 /**
- * Gets the hex color part of an alpha hex color. For example, from '#abcdef55'
- * return '#abcdef'.
+ * Gets the hex or alpha color part of an alpha hex color
  * @param {string} colorWithAlpha The alpha hex color to get the hex color from.
+ * @param {number} startIndex The start index of the alpha hex color to strip
+   from.
+ * @param {number} endIndex The end index of the alpha hex color to strip until.
  * @return {string} The hex color where the alpha part has been stripped off.
  */
-goog.color.alpha.extractHexColor = function(colorWithAlpha) {
+goog.color.alpha.extractColor = function(colorWithAlpha, startIndex, endIndex) {
```

Comment:
private methods should be named with a trailing underscore:
goog.color.alpha.extractColor_

thanks.